roachtest,admission: assume ownership of adjacent tests

irfansharif opened the pull request:
Release note: None

irfansharif committed:
roachtest,admission: assume ownership of adjacent tests

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/89027)
<!-- Reviewable:end -->

irfansharif committed:
roachtest: delete tpccbench/kv variants that disable AC

Admission control has been enabled as default for multiple releases now,
these roachtest variants that disable them aren't giving us value. If
there are regressions observed in the standard kv/tpccbench tests that
end up being attributable to admission control, we can learn about it
when investigating the regression directly instead of needing to compare
it to test runs where AC is disabled.

Release note: None

irfansharif committed:
roachtest: reduce frequency of admission tests to weekly cadence

We'll be adding lots of these tests and it's excessive to be running
them nightly. These tests will also be reframed as benchmark tests with
no explicit PASS/FAIL since we're not verifying cluster stability, but
instead the degree to which latency/throughput is protected in the
presence of resource saturation. There too a nightly cadence feels
unnecessary to track regressions.

Release note: None

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 7 of 7 files at r1, 2 of 2 files at r2, 2 of 2 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/89027#-:-NDT6y3t0WwwwR0MUfTm:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @irfansharif)

___
*[`pkg/cmd/roachtest/tests/admission_control_multi_store_overload.go` line 26 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/89027#-NDT5kZtBbW6ePfTQ7V8:-NDT5kZtBbW6ePfTQ7V9:b-x9mnvj) ([raw file](https://github.com/cockroachdb/cockroach/blob/8966a2cefeb7f20c6b1b8f67e27b7b7782b1657b/pkg/cmd/roachtest/tests/admission_control_multi_store_overload.go#L26)):*
> ```Go
> )
> 
> func registerMultiStoreOverload(r registry.Registry) {
> ```

I assumed this is just code movement so didn't look at the details.


<!-- Sent from Reviewable.io -->

irfansharif reviewed (commented):
bors r+

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/89027#-:-NDTCtIA1shdrceSJFrU:b-ydi9ay)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @sumeerbhola)

___
*[`pkg/cmd/roachtest/tests/admission_control_multi_store_overload.go` line 26 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/89027#-NDT5kZtBbW6ePfTQ7V8:-NDTCuNhDo6vS6jI474h:bpzhugl) ([raw file](https://github.com/cockroachdb/cockroach/blob/8966a2cefeb7f20c6b1b8f67e27b7b7782b1657b/pkg/cmd/roachtest/tests/admission_control_multi_store_overload.go#L26)):*
<details><summary><i>Previously, sumeerbhola wrote…</i></summary><blockquote>

I assumed this is just code movement so didn't look at the details.
</blockquote></details>

It was.


<!-- Sent from Reviewable.io -->
